rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -631,9 +632,12 @@ def _get_transaction_receipt(
             self,
             web3: Optional[Web3],
             tx_hash: EVMTxHash,
-    ) -> dict[str, Any]:
+    ) -> Optional[dict[str, Any]]:
         if web3 is None:
             tx_receipt = self.etherscan.get_transaction_receipt(tx_hash)
+            if tx_receipt is None:
```

Comment:
Have you tested that this is possible and that for a non-existing transaction this returns `None` ?